web/src/components/notification-banner/notification-banner.tsx

```diff
@@ -42,28 +52,27 @@ export default function NotificationBanner({
   const [show, setShow] = useState(false);
 
   const el = useRef(null);
+  const { storageKey = '' } =
+    notification.kind == 'banner' && notification.bannerProps;
 
   useEffect(() => {
     setShow(true);
   }, []);
 
+  function hideBanner(storageKey?: string) {
+    setShow(false);
+    storageKey && localStorage.setItem(storageKey, JSON.stringify(true));
```

Comment:
Not blocking, but `localStorage` access should be behind a `useEffect`. Check out https://joshwcomeau.com/react/persisting-react-state-in-localstorage/ for more info

Reply:
TIL! Thank you!